### ISSUE TITLE
Unable to create zLinux VM for rhel7 (s390x) on redhat 8 host os

### ISSUE THREAD

**Issue by amitbatajoo** [hercules-390/hyperion#293]
Hello,
I tried to boot rhel 7(s390x) on Redhat 8 as Host OS under  Hercules 4.4.1.
Actually I am trying
RHEL8(host OS)---------->Hyperion tool(Hercules4.4.1) ----------> z/VM ---------------> RHEL 7(s390x)

I failed to complete the boot setup. Every time there is an error as below:

17:35:10    127.616005! RPC: Registered tcp transport module.
17:35:10    127.616074! RPC: Registered tcp NFSv4.1 backchannel transport module.
17:35:13   32m  OK   0m! Started dracut pre-udev hook.
17:35:13          Starting udev Kernel Device Manager...
17:35:13   32m  OK   0m! Started udev Kernel Device Manager.
17:35:13    130.961177! systemd-udevd 472!: starting version 208
17:35:13          Starting dracut pre-trigger hook...
17:35:17   32m  OK   0m! Started dracut pre-trigger hook.
17:35:17          Starting udev Coldplug all Devices...
17:35:18   32m  OK   0m! Started udev Coldplug all Devices.
17:35:18          Starting dracut initqueue hook...
17:35:18          Starting Show Plymouth Boot Screen...
17:35:18   32m  OK   0m! Reached target System Initialization.
17:35:18          Mounting Configuration File System...
17:40:15 Warning: /dev/root does not exist
17:40:15
17:40:15 Generating "/run/initramfs/rdsosreport.txt"
17:40:16
17:40:16
17:40:16 Entering emergency mode. Exit the shell to continue.
17:40:16 Type "journalctl" to view system logs.
17:40:16 You might want to save "/run/initramfs/rdsosreport.txt" to a USB stick or /boot
17:40:16 after mounting them and attach it to a bug report.
17:40:16
17:40:16
17:40:16 dracut:/#
The configuration is as below:
ARCHLVL   z/Arch

FACILITY  ENABLE BIT37    # FPEXT   floating point extension
#  Needed to enable SUSE SLES 12 under
FACILITY  ENABLE BIT44    # PFPO    PFPO instruction  #  Hercules version 4.00 only
FACILITY  ENABLE BIT45    # FAST_BCR_SERIAL
         #  Needed for Ubuntu 16.04.3
########################################################################
FACILITY  ENABLE BIT48    # DFPZC decimal-floating-point zoned-conversion
FACILITY  ENABLE BIT49    # MISC
                          #  Needed for Ubuntu 16.04.3 like BIT45
########################################################################
CNSLPORT  3270
CPUMODEL  3090            # 2066=z900, 2098=z10, 2818=z196, 2828=zEC12
CPUSERIAL 002623
NUMCPU    2
MAXCPU    2
DIAG8CMD ENABLE
ECPSVM   YES
 
LPARNAME  RHEL
LPARNUM   18                   # LPARNUM = SA number
 
MAINSIZE  2048
MOUNTED_TAPE_REINIT  DISALLOW
OSTAILOR  LINUX
PANRATE   SLOW
SHCMDOPT  ENABLE     NODIAG8
 
# Display Terminals
0700 3270
0701 3270
 
# dasd
#0120 3390 ./dasd/ubuntu.120.cckd
#0120 3390 /opt/rhel-hercules/dasd/root.0120
#0121 3390 /opt/rhel-hercules/dasd/swap.0121
0120  3390 /opt/rhel-hercules/dasd/sys1.dasd
0121  3390 /opt/rhel-hercules/dasd/sys2.dasd
 
# tape
0581    3420
 
# network                               s390     realbox
0A00,0A01  CTCI -n /dev/net/tun -t 1500 192.168.122.2 192.168.122.1

Does anyone know the correct configuration to boot and setup Rhel7 (s390x) using Hercules.

Please suggest me the solution.
thank you in advance.

**Comment by wrljet** [comment]
Does *Closed* mean you got it working?

**Comment by amitbatajoo** [comment]
@wrljet No, using Hercules its not working till now. It seems really difficult to Installing rhel7 Server s390x in Hercules. 
Is there any hope for working tutorial on it?
